docs/testrunner-toolkit/configuration/testcafe.md

```diff
@@ -65,13 +106,101 @@ __Example__:
   - name: "saucy test"
 ```
 
-### `match`
+### `browserName`
+
+__Description__: Name of desired browser. Although Testcafe supports triggering one test in multiple browsers, it'd better here to split them into every suite to indicate each suite has its own test point.
+
+__Type__: *string*
+
+__Example__:
+```yaml
+  browserName: "chrome"
+```
+
+### `src`
+
+__Description__: The explicit name, file glob, or location of the test files
+
+__Type__: *object*
+
+__Example__:
+```yaml
+  src:
+    - "tests/test_file1.test.js"
+    - "tests/integrations"
+    - "*/*.test.js"
+```
+
+### `env`
+
+__Description__: Environment variables. It can be expanded.
+
+__Type__: *object*
+
+__Example__:
+```yaml
+  env:
+    hello: world
+    foo: $my_bar
+```
+
+### `screenshots`
+
+__Description__: Native screenshots settings for testcafe.
+
+__Type__: *object*
+
+__Example__:
+```yaml
+  screenshots:
+    takeOnFails: true
+    fullPage: true
+```
+
+### `speed`
+
+__Description__: Specifies the test execution speed. Tests are run at the maximum speed by default. You can use this option to slow the test down. Provide a number between 1 (the fastest) and 0.01 (the slowest).
+
+__Type__: *float64*
+
+__Example__:
+```yaml
+  speed: 1
+```
+
+### `platformName`
+
+<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small><a href="/testrunner-toolkit/running-tests#test-on-sauce-labs">ℹ</a></p>
+
+__Description__: Operating system on which the browser and test runs.
+
+__Type__: *string*
+
+__Example__:
+```yaml
+    platformName: "Windows 10"
+```
+
+### `testMatch`
+
+__Description__: The explicit name, regex, or location of the test files
+
+__Type__: *string* | *regex*
+
+__Example__:
+```yaml
+    testMatch: '**/*.js'
+```
+
+### `screenResolution`
+
+<p><small><Highlight color="#ad1415">sauce cloud only</Highlight></small><a href="/testrunner-toolkit/running-tests#test-on-sauce-labs">ℹ</a></p>
 
-__Description__: Regular expression for locating test files.
+__Description__: Field where you can change the browser window screen resolution.
```

Comment:
Maybe we can shorten this to just `Set browser window screen resolution`

Reply:
updated